homeassistant/components/apsystems/__init__.py

```diff
@@ -12,15 +14,23 @@
 
 PLATFORMS: list[Platform] = [Platform.SENSOR]
 
-ApsystemsConfigEntry = ConfigEntry[ApSystemsDataCoordinator]
+ApsystemsConfigEntry = ConfigEntry["ApSystemsData"]
```

Comment:
Side note: you are mixing up casing quite a lot in the integration:
`Apsystems`, `APsystems`, `ApSystems`, ...
I would recommend to keep it aligned with the base library, which seems to always use `APsystems` prefix.

Reply:
The APsystems is the correct usage, but ApSystems is UpperCamelCase imo.

Reply:
This is not resolved.
If you wish to use `ApSystems`, then it's ok, but it should be used here:
`ApSystemsConfigEntry` not `ApsystemsConfigEntry`